test/utils/test_tree.py

```diff
@@ -14,17 +29,20 @@ def test_build_graph(self):
 
         g2 = build_graph(t, graph_title="Demo 2 of build_graph.")
         assert "Demo 2 of build_graph." in g2.source
-        assert "<root_0>" in g2.source
-        assert "<root_0>.x" in g2.source
+        assert "root_0" in g2.source
         assert len(g2.source) <= 580
 
         g3 = build_graph((t,), graph_title="Demo 3 of build_graph.")
         assert "Demo 3 of build_graph." in g3.source
-        assert "<root_0>" in g3.source
-        assert "<root_0>.x" in g3.source
+        assert "root_0" in g3.source
```

Comment:
why do we delete assertion about `root_0.x`

Reply:
Because `<` and `>` are escaped with html syntax in new version, so native `<root>` substring will not appear in new svg code.